src/lib/banal/Struct.cpp

```diff
@@ -150,7 +151,7 @@ using namespace std;
 #define DEBUG_SKEL_SUMMARY  0
 
 #if DEBUG_CFG_SOURCE || DEBUG_MAKE_SKEL || DEBUG_SHOW_GAPS
-#define DEBUG_ANY_ON  1
+#define DEBUG_ANY_ON  0
```

Comment:
Other side of the conditional is identical. Do you mean to delete the `DEBUG_ANY_ON` usage?